benchmarking/README.md

```diff
@@ -0,0 +1,45 @@
+# WebAssembly Benchmarking Subgroup Charter
+
+The Benchmarking Subgroup is a sub-organization of the [WebAssembly Community
+Group](https://www.w3.org/community/webassembly/) of the W3C. As such, it is
+intended that its charter align with that of the CG. In particular, the sections
+of the [CG charter](https://webassembly.github.io/cg-charter/) relating to
+[Community and Business Group
+Process](https://webassembly.github.io/cg-charter/#process), [Contribution
+Mechanics](https://webassembly.github.io/cg-charter/#contrib),
+[Transparency](https://webassembly.github.io/cg-charter/#transparency), and
+[Decision Process](https://webassembly.github.io/cg-charter/#decision) also
+apply to the Subgroup.
+
+## Goals
+
+The mission of this subgroup is to provide a forum for collaboration on
+benchmarking efforts for WebAssembly programs, sharing best practices, tools,
+and knowledge related to benchmarking.
+
+## Scope
+
+The Subgroup will consider topics related to benchmarking of WebAssembly, including:
+
+- Best practices for benchmarking WebAssembly
+- Development and sharing of benchmarking tools
+- Knowledge sharing about existing benchmarks, systems, and projects
+- Public, high-level, recommendations about best practices, frameworks, suites, metrics, for WebAssembly benchmarking
+
+## Deliverables
+
+### Non-normative reports
+
+The Subgroup may produce non-normative material such as requirements documents,
+recommendations, and use cases related to benchmarking.
+
+### Software
+
+The Subgroup may produce software related to benchmarking of WebAssembly, including but not limited to:
+
+- Benchmarking tools specifically designed for WebAssembly
+- Libraries or frameworks to aid in the benchmarking of WebAssembly applications
```

Comment:
Shouldn't actual benchmarks be included in this list? Or is that what is meant by "benchmarking tools"? (If so, I don't think that's clear.)

Reply:
I think it's OK to include it, though I don't think we'll primarily be producing benchmarks, as there are certainly caveats to any we do produce, and there are many out there already. Distilling and collating might fit here.